yaml-solutions/01. challenge-02/captureorder-deployment.yaml

```diff
@@ -33,10 +33,7 @@ spec:
               cpu: "500m"
           env:
           - name: MONGOHOST
-            valueFrom:
-              secretKeyRef:
-                name: mongodb
-                key: mongoHost
```

Comment:
Need to validate the DevOps pipeline https://github.com/Azure/azch-captureorder/blob/master/azure-pipelines.yml#L69

Reply:
No longer an issue since I revert other changes